.pre-commit-hooks.yaml

```diff
@@ -28,3 +28,9 @@
     entry: ' `[^`]+[^_]`([^_]|$)'
     language: pygrep
     types: [rst]
+-   id: python-noeval
+    name: Check for eval()
+    description: 'A quick check for `eval()` built-in function'
+    entry: '\beval\s*\((.*|\s*\))'
+    language: pygrep
+    types: [python]
```

Comment:
please alphabetize

Also the regex can be much simpler:

```
\beval\(
```

(also if you want to add some tests (optional), there's a little harness in `tests/hooks_test.py` -- might be good to show that `eval("foo")` gets matched and `ast.literal_eval(bar)` does not for instance)

Reply:
It looks like I didn't need to update the Readme as well.

Reply:
yeah there's a pre-commit hook which will put the appropriate text there -- `pre-commit run --all-files` will put the README into good shape